techhive/models/messageModel.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+
+const MessageSchema = mongoose.Schema(
+    {
+     message: {
+        text: {type: String, required: true},
+    },
+    users: Array,
```

Comment:
Based on the implementation of `addMessage`, `users` is an array of sender/receiver usernames: `[from, to]`. 
Why do we also need the `sender` in the message schema?